DH_Effects/Classes/DH_MolotovMark.uc

```diff
@@ -0,0 +1,24 @@
+//==============================================================================
+// Darkest Hour: Europe '44-'45
+// Darklight Games (c) 2008-2019
+//==============================================================================
+
+
+class DH_MolotovMark extends BlastMark;
+
+defaultproperties
+{
+	ProjTexture=Texture'DH_Weapon_tex_molotov.molotov_mark'
+	MaterialBlendingOp=PB_Add
+	FrameBufferBlendingOp=PB_AlphaBlend
+	DrawScale=0.5
+	bGameRelevant=true
+	PushBack=24
+	LifeSpan=30
+	FOV=1
+	MaxTraceDistance=60
+	bProjectBSP=true
+	bProjectTerrain=true
+	bProjectStaticMesh=true
+	bProjectActor=true
```

Comment:
`bProjectActor` should be set `false`. The projection is frozen in space; if the actor moves, you will see the blast mark sliding over it.

Reply:
Fixed in commit 857987d72bdaaae70274c307dd6c8b0b05d2f1e2.